### ISSUE TITLE
VarDictJava 1.5.5: large deletions reported starting out of region bounds

### ISSUE THREAD

**Issue by chapmanb** [AstraZeneca-NGS/VarDictJava#142]
We've been testing the new 1.5.5 release with `--nosv` turned on to just report SNPs and indels. We're running into an issue where VarDictJava will report larger deletions starting outside of the calling region bounds. For example, when calling on this region:
```
chr1    150578117       150578617       MCL1
```
There is a large deletion reported in the output that starts at 150577490 before the start coordinates:
```
chr1    150577490       .       TCCTAGAAAACAAAAAAGAAAAGCACATTTTCAAGTATGGGTTTCTGCTCTCTTACAACACTACTATCCAGAGAGAAGGTAAATTAAAATATCTAAGGTTTCCCCCTAAAGCAATAGTAAACCAATTATTATTCACCCCGGGGCAAAAAAAGAAAATAAGTTCTTCAGTTTCTTAAGCGGGGACCCATGATAATTAAACTTCATTTTACAGTACTATCTTGGTCTTATTAGGATTTATCCTAACAAAAGGACTTCCAGAGTTCCCATGAGGTAAAAGACGAAAACCCTTATTCTCAAATCTCCACCACCTAATATTTCAGTAACGCCTCAATTGCGCAGATCAGTTTTTCTCAACGTAGTTAAGTACATCCTGCATCAGATCTGGGTTTGTTTAAAACGCAGGCTCCACCCCATATCTTCTCCATTAAAACAGGCACCGCCTTAGGAATTATTTCAGCTATCCTCCCCTGGTGACTTATTTTTTCATGGTTTGAATCCACTGAAGAGTATTCAACTGAATTGGAACAAAAAATTGCCTAGAGAAAACCAAGATTTTTGATACAATTCAAGACCGCCATACCACTATCTCTAAGGATCCACTACTTAAATCAACATCAAATAAACAATGGTCCTTTAGTTAGAGCCTGCAAACAGCCGTGCGTCATAAAAACCTTTAGATATCCCCACCTCTCTAAAAAAAATCCTTCATTCTACTTCCACTCCAAGGCCCCTTTCATCCTTAAGGCAAACTTA       T       468     PASS    SAMPLE=PrCa-11-PROSTATE-N-1-IDTp7UMI35-GGCTCAAT;TYPE=Deletion;DP=3528;VD=1377;AF=0.3903;BIAS=2:2;REFBIAS=1307:1553;VARBIAS=768:609;PMEAN=37.2;PSTD=1;QUAL=44.9;QSTD=1;SBF=0;ODDRATIO=1.49826201606137;MQ=60;SN=2754;HIAF=0.3257;ADJAF=0.3903;SHIFT3=2;MSI=1;MSILEN=1;NM=0.1;HICNT=1377;HICOV=4228;LSEQ=GGAAGAACTCCACAAACCCA;RSEQ=CCCAGCCTCTTTGTTTAACT;DUPRATE=0;SPLITREAD=713;SPANPAIR=0        GT:DP:VD:AD:AF:RD:ALD   0/1:3528:1377:2860,1377:0.3903:1307,1553:768,609
```
This causes issues when running in parallel over different regions, as this event ends up getting reported twice: once at the end of the previous region file, and once in this one. This leads to out of order and duplication errors when merging.

Would it be possible to restrict output to strictly start in regions of interest or remove these larger events if they come from sv calling (since `--nosv` is specified on the command line)?

I'll send offline a reproducible example that demonstrates the issue. Thanks so much for looking at this.

**Comment by PolinaBevad** [comment]
@chapmanb, hello!
It is fixed in PR #149. Test case was added to set.
We added the option `--deldupvar` that will skip variants started outside the region of interest (SVs and long indels) to avoid duplications. 
Thank you for reporting this!

**Comment by PolinaBevad** [comment]
@chapmanb, hello!
I will close this issue because the fix was released since 1.5.6 version.
Thank you again!
